README.md

```diff
@@ -0,0 +1,146 @@
+# NCEPLIBS
+
+## Warning
+
+This repository and the documentation are rapidly changing and not ready for general community use.
+
+## Introduction
+
+This is a compilation of libraries known as NCEPLIBS that are required for building and running the UFS weather model. For general information about NCEPLIBS and the UFS weather model, the user is referred to the [Wiki](https://github.com/NOAA-EMC/NCEPLIBS/wiki).
+
+It includes the following libraries:
+
+| NCEP library         | Version                                               |
+|----------------------|-------------------------------------------------------|
+| NCEPLIBS-bacio       | 2.1.0                                                 |
+| NCEPLIBS- bufr       | 11.3.0                                                |
```

Comment:
Unwanted space between NCEPLIBS- and bufr. Strictly speaking bufr is not required for UFS (at least for version 1), and I think nemisogfs also is not required.

Reply:
I'll fix the space, the other aspect of what is required or not will probably be touched upon at today's NCEPLIBS meeting ...